packages/CMS/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@canner/cms",
```

Comment:
This should be `canner` not `@canner/cms`.

And the `package.json` of this package should be removed like gatsby https://github.com/gatsbyjs/gatsby/blob/master/package.json

Reply:
I've changed it!